models.py

```diff
@@ -19,11 +19,12 @@
 """
 
 
+
 class Constants(BaseConstants):
     name_in_url = 'DecisionStudy'
-    # players_per_group = 9 # value for production
-    players_per_group = 2 # only for testing
-    num_rounds = 2
+    num_rounds = 1
+    players_per_group = 9
+
```

Comment:
We should still have some commented line with `players_per_group = 2 # only for testing`, as this is required to run the Test session in the experiment